lib/resources/example_json/r4_examples_document_reference.rb

```diff
@@ -4,168 +4,391 @@ module Cerner
-                        'valueCode': 'unknown',
-                        'url': 'http://hl7.org/fhir/StructureDefinition/data-absent-reason'
-                      }
-                    ]
+                  'code': '11506-3',
+                  'userSelected': false
+                }
+              ],
+              'text': 'Progress Note Generic'
+            },
+            'category': [
+              {
+                'coding': [
+                  {
+                    'system': 'http://loinc.org',
+                    'code': '11506-3',
+                    'userSelected': false
                   }
+                ]
+              },
+              {
+                'coding': [
+                  {
+                    'system': 'http://hl7.org/fhir/us/core/CodeSystem/us-core-documentreference-category',
+                    'code': 'clinical-note',
+                    'display': 'Clinical Note',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Clinical Note'
+              }
+            ],
+            'subject': {
+              'reference': 'Patient/12742400',
+              'display': 'PETERS, TIM A'
+            },
+            'author': [
+              {
+                'reference': 'Practitioner/12742069',
+                'display': 'Portal, Portal'
+              }
+            ],
+            'authenticator': {
+              'reference': 'Practitioner/12742069',
+              'display': 'Portal, Portal'
+            },
+            'content': [
+              {
+                'attachment': {
+                  'contentType': 'application/pdf',
+                  'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197411793',
+                  'title': 'Auto TC Summary',
+                  'creation': '2020-12-10T07:19:53.000Z'
+                }
+              }
+            ],
+            'context': {
+              'encounter': [
+                {
+                  'reference': 'Encounter/97953530'
+                }
+              ],
+              'period': {
+                'end': '2020-12-10T07:19:53.000Z'
+              }
+            }
+          }
+        }
+      ]
+    }.freeze
+
+    R4_DOCUMENT_REFERENCE_PATIENT_ACCESS_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '941ab54e9-4501-446a-9cee-6cb14c37dab3',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411797',
+          'resource': R4_DOCUMENT_REFERENCE_PATIENT_ACCESS
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411793',
```

Comment:
Url should be of closed endpoint not open
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400

Reply:
addressed